assets/src/edit-story/app/prepublish/components/videoOptimization.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  margin-top: 10px;
+`;
+
+const Thumbnail = styled.img`
+  height: 66px;
+  width: 44px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  object-fit: cover;
+  object-position: 50% 50%;
+`;
+
+export function VideoOptimization({ element }) {
+  return (
+    <Container>
+      <Thumbnail
+        src={element.resource?.poster}
+        alt={element.resource?.alt}
```

Comment:
do you think a fallback would be good here?

Reply:
Do we have a fallback image or icon in the repo rn? or do I have to go ask design for one?

Reply:
ah i meant a fallback alt

Reply:
(i don't think we have a fallback image, we can follow up with that)

Reply:
okee. I'll add a fallback alt rq

Reply:
alright, should be good now